python/pyarrow/table.pxi

```diff
@@ -4708,6 +4711,11 @@ cdef class Table(_PandasConvertible):
 
         return table
 
+    drop = drop_columns
+    """
+    Alias of Table.drop_columns, kept for backwards compatibility.
+    """
```

Comment:
Is this docstring for sphinx? (interactively checking the docstring in a terminal/notebook, that still gives the `drop_columns` one)

Reply:
Great catch!! Will fix this.

Reply:
I'm going to remove the alias and use a wrapper. I'm not sure if it's possible to update the docstring of an alias without affecting the original method's docstring.

I tried this:
```
    drop = drop_columns
    drop.__doc__ = """Alias of Table.drop_columns, kept for backwards compatibility."""
```
..but got an error:
```
Traceback (most recent call last):
  ...
  File "pyarrow/table.pxi", line 4729, in init pyarrow.lib
AttributeError: attribute '__doc__' of 'method_descriptor' objects is not writable
```

Reply:
I suppose that's because it's implemented in cython, and then that attribute is not writeable as it would be in python (just a guess, didn't actually verify)

But anyway, the wrapper looks good!